configure.ac

```diff
@@ -46,5 +46,12 @@ AC_TRY_LINK([
 AC_DEFINE(HAVE_FULL_QUEUE_H,, Define if sys/queue.h includes required macros)
 ])
 
+AC_TRY_LINK([
+	#include <ncursesw/curses.h>
+	#include <ncursesw/term.h>
+], [], [
+AC_DEFINE(HAVE_NCURSESW_H,, Define if ncursesw is available)
+])
```

Comment:
I'm just using it do define the constant `HAVE_NCURSESW_H` in order for the compiler to decide what it should link.